polytope/polytope.py

```diff
@@ -310,7 +310,7 @@ def rotation(self, u, v, theta=None):
         (1) u and v are the indicies (0, N] of two basis vectors, and theta is
         the angle of rotation.
 
-        (2) u and v are two unit vectors, angle of rotation is the angle
+        (2) u and v are two vectors, angle of rotation is TWICE the angle
```

Comment:
Can you provide citations? However, this might not be important because it is not implemented yet.